fmralign/alignment_methods.py

```diff
@@ -82,6 +83,16 @@ def optimal_permutation(X, Y):
     return permutation
 
 
+def projection(x, y):
+    return np.dot(x, y) / np.linalg.norm(x)**2
+
+
+def voxelwise_signal_projection(X, Y, n_jobs):
+    return Parallel(n_jobs)(delayed(projection)(
```

Comment:
dosctring here too. Also, as it lay not be part of the public API, you may prepend an undrescore:
_voxelwise_signal_projection
Same for the previous function.

Reply:
Actually, I'm wondering: you want to directly apply that at the matrix level, avoiding the loop.

Reply:
After doing the benchmark, this is 4x faster this way:
```
from fmralign.alignment_methods import _voxelwise_signal_projection
import numpy as np
X = np.random.randn(100, 500000)
Y = np.random.randn(100, 500000)
```


`%timeit -n 100 _voxelwise_signal_projection(X, Y, 1)`
> 100 loops, best of 3: 109 ms per loop

`%timeit - n 100 np.sum(X * Y, 1) / np.sum(X * X, 1)`
 > 100 loops, best of 3: 703 ms per loop